publication-doi-commons/src/main/java/no/unit/nva/publication/doi/dto/Publication.java

```diff
@@ -88,6 +105,19 @@ public PublicationDate getPublicationDate() {
         return publicationDate;
     }
 
+    /**
+     * Flag used during EventBridge pattern matching.
+     *
+     * @return  true if modifiedDate is same as doiRequest.modifiedDate
+     */
+    @JsonProperty("sameModifiedDateForDoiRequest")
+    public boolean isSameModifiedDateForDoiRequest() {
```

Comment:
I imagine we probably want to emit events for when Publication modifiedTimestamp is newer then the modifiedDoiRequestTimestamp as well, to ensure metadata is always up2date.

Reply:
I think it necessary to consider DoiRequests as different entities in the Publications table. Re-design of the publications table will come soon.

Reply:
This is out of scope for this task.